packages/dcos-integration-test/extra/test_ucr.py

```diff
@@ -0,0 +1,229 @@
+import uuid
+
+TEST_APP_NAME_FMT = 'integration-test-{}'
+
+
+def test_if_ucr_app_can_be_deployed_with_image_entrypoint(dcos_api_session):
+    """Marathon app deployment integration test using the Mesos Containerizer.
+
+    This test verifies that a marathon ucr app can execute a docker image
+    default entrypoint.
+    """
+    test_uuid = uuid.uuid4().hex
```

Comment:
@Gilbert88 , all these app definitions can be refactored 'out' a command `base_app` outside of these functions.

You could create `base_app` definition with good known defaults

```
base_app =  {
'id': TEST_APP_NAME_FMT.format(test_uuid),
'cpus': 0.1,
....
}
```
and each of the `test_` functions.

You could, then

```
app = copy.deepcopy(base_app)
app['container']['docker] = {'image': 'mesosphere/inky'} # or other similar modifcations
and use the app.
```

Each of the test code will be smaller.

However, I have another important question. Why are testing "images deployments"? How does images **exercise** any functionality of  base dcos or mesos code? I could not figure this out even after reading the docstrings of these methods.